### DISCUSSION TITLE
Nzremix mod nzp psp android pc by MR HN STUDIOS (YOxdxd)

### DISCUSSION THREAD

**Discussion by Mrhn504** [nzp-team/nzportable#851] [Other Content Releases]
![Screenshot_2024-07-18-20-45-51-577](https://github.com/user-attachments/assets/dc8d6bc5-609c-49c3-87e6-f63d5ebb7d69)
![Screenshot_2024-07-18-21-05-39-815](https://github.com/user-attachments/assets/bf3f3f5c-e7f8-4554-a9fc-d272897b8a51)
![1721229406742](https://github.com/user-attachments/assets/1cf7941b-f702-41e5-9110-ddcd920f1932)

**Comment by Mrhn504** [comment]
![img_1_1716914581310](https://github.com/user-attachments/assets/a5c4fe29-bd97-4806-ae91-e4cab8c97198)
![img_3_1716509076064](https://github.com/user-attachments/assets/e6438387-3c88-4b81-bc72-202ba373d67f)
![img_1_1716509044893](https://github.com/user-attachments/assets/89cbee1b-0ab3-4d70-9d90-2e1518e76842)
![img_6_1717280796106](https://github.com/user-attachments/assets/daa5fcf8-58e6-46ac-9f6c-b226a160f24b)

Aún lo desarrollo pero muy pronto sacaría unas versiones de prueba😄

**Comment by Riuk1z** [comment]
me puedes explicar como va a hacer el mod?

**Comment by Mrhn504** [comment]
El mod saldría para la plataforma de PC android y PSP el mod se basa en
adaptar cosas como modelos y mapas con mejor calidad. Y combinar cosas de
otros cod zombies

**Comment by endis-offical** [comment]
what is this? This looks like a severly bugged out version of town with CS source arms?

**Reply by Mrhn504** [reply]
https://github.com/nzp-team/nzportable/discussions/1301
Hola está discusion del proyecto que empeze fue hace 2 años pero la última actualización que subí fue hace unos meses actualmente lo tengo en pausa la idea era hacer un mod que fuera mas similar a cod con modelos igual a los originales pero es demasiado trabajo 😅 te invito a probarlo por si gustas te dejo el enlace a la discusión mas reciente esta compilado en un solo archivo .PAK
